test/utils/configUtils.js

```diff
@@ -1,11 +1,13 @@
-function getServerlessConfig (serverless = {}) {
+const path = require('path');
+
+function getServerlessConfig(serverless = {}) {
   return {
-    getProvider: serverless.getProvider || (() => {}),
+    getProvider: serverless.getProvider || (() => { }),
```

Comment:
Why this space?

Reply:
I thought I removed all that auto reformat nonsense before the commit, guess some snuck in anyway. I'll get that out of the PR.